packages/layout/src/list.tsx

```diff
@@ -127,6 +146,7 @@ export const ListIcon = forwardRef<IconProps, "svg">(function ListIcon(
       display="inline"
       verticalAlign="text-bottom"
```

Comment:
Moving the default styles into the theme object would improve autocompletion when extending the theme and we would follow more the separation of concerns. The components just apply styles and manage their state and the styling is maintained in the theme.

Reply:
Yeah we are all over the place with this. I wanted to chat with Sage about this at some point